src/doc/trpl/no-stdlib.md

```diff
@@ -38,6 +38,7 @@ fn start(_argc: isize, _argv: *const *const u8) -> isize {
 // provided by libstd.
 #[lang = "stack_exhausted"] extern fn stack_exhausted() {}
 #[lang = "eh_personality"] extern fn eh_personality() {}
+#[no_mangle] pub extern fn rust_eh_unwind_resume() {}
```

Comment:
Without it doctests fail on Windows.  Also, if we ever drop libgcc on other platforms, we'll need it.